src/tcgwars/logic/impl/gen4/SecretWonders.groovy

```diff
@@ -341,18 +341,33 @@ public enum SecretWonders implements LogicCardInfo {
           pokePower "Telepass", {
             text "Once during your turn , you may search your opponent’s discard pile for a Supporter card and use the effect of that card as the effect of this power. (This Supporter card remains in you opponent’s discard pile.) You can’t use more than 1 Telepass Poké-Power each turn. This power can’t be used if Gardevoir is affected by a Special Condition."
             actionA {
-            }
-          }
+              checkLastTurn()
+              assert bg.em().retrieveObject("Telepass") != bg.turnCount : "You can't use more than 1 Telepass Poke-Power each turn."
```

Comment:
You need to store this object once the pokePower is used as well. As it is you will be retrieving a value that never equates to `bg.turnCount` no matter how many Gardevoir use the Poke Power.

Reply:
Hopefully fixed

Reply:
There's actually a `checkNoSPC()` usually used for Pokémon Powers

Reply:
Also, should probably unindent some of this now that it isn't in an if block.